### ISSUE TITLE
Feature Request - Please add the option to select a target disk for installation

### ISSUE THREAD

**Issue by sven-borkert** [linuxmint/ubiquity#79]
Hi,

please add the option to select a target disk when using the option "Erase disk and install Linux Mint"

![grafik](https://github.com/linuxmint/ubiquity/assets/10104966/c26bcc21-7bc0-43d5-aca0-589648425f43)

I know I can go to "Something else" and do it all manually.

But I actually love the comfort of the installer just creating everything with LVM and encryption, without me having to fiddle around in that little window (cannot even be resized) with all my disks and create it all manually. It would be a massive improvement for me, if I could just select a disk here, that the installer will complely use without touching any of the other disks.

Is there any other way to achieve this? Currently I just unplug all other disks for a Mint installation. Every year I come back checking if the installer has been improved, but I seems I'm the only one who would find that option useful?

Regards,
Sven

**Comment by Penguinjumper** [comment]
Yes this is just crazy. Everyone says "use mint, it's super easy, don't bother with win11". Then here I go, wanting to install it on my second drive to try it out and I have to spend an hour figuring out how to manually configure drives to run with mint. And that's not even including the 2nd hour because I want full disk encryption just like with bitlocker on windows. 

It's these kind of things that really stop me from recommending linux to not-so-technical persons around me...

**Comment by Plinsboorg** [comment]
https://github.com/linuxmint/ubiquity/issues/95
Yes please

**Comment by gdurif** [comment]
@Delocca I just tried today and after selectring the "Erase disk and install Linux Mint", the installer did NOT ask me on which disk.

Generally speaking, as stated by previous commenters, it would be very useful to be able to select on which disk do a standard install, because computer may have several disk (e.g. SDD + HDD) sometimes difficult to unplug (e.g. laptop) and you may have valid reasons for not letting the system decide at random (based on detected device order) which disk it is going to erase (other system or data on other disks).
